### DISCUSSION TITLE
Version 2 support

### DISCUSSION THREAD

**Discussion by jolivaSan** [panva/jose#189] [Q&A] [answered]
Hi @panva How many time do you plan to support version 2? i mean for security updates etc...

**Comment by panva** [comment, marked as answer]
You can see that in the readme.

**Reply by jolivaSan** [reply]
Ups so sorry to ask, thanks

**Reply by panva** [reply]
No worries. As seen with the latest advisory i even updated 1.x. So long as it is possible vulnerability fixes will be coming out.
